client/src/Components/Header/Header.js

```diff
@@ -13,13 +13,11 @@ const Header = () => {
           <header>
             <nav>
               <Link to='/' className='logo'>BEAM</Link>
-              <Link to='/how-you-felt' className='uni-btn'>Feelings Archive</Link>
```

Comment:
GO ON, 'GIT!

Reply:
I don't remember exactly, but could we get rid of the `<Switch>` and other logic now that we only have one route in the Nav?

Reply:
Issue added!